source/preferences/manage-your-web-notifications.rst

```diff
@@ -0,0 +1,77 @@
+Manage your web notifications
+==============================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+.. |chrome-mention-badge| image:: ../images/chrome-mention-badge.png
+  :alt: An example of a Chrome web browser mention badge indicator.
+
+.. |chrome-activity-badge| image:: ../images/chrome-activity-badge.png
+  :alt: An example of a Chrome web browser activity badge indicator.
+
+Enable notifications
+--------------------
+
+From Mattermost v9.10, Mattermost prompts you to enable notifications in your web browser.
```

Comment:
Suggesting we word this as `grant permissions` given that is also how it's worded in the banner in-product shown in the screenshot. It would also help if they later need to find the permissions on their own in the browsers 

https://github.com/mattermost/docs/pull/7378#discussion_r1765574709